lib/resources/r4/appointment.yaml

```diff
@@ -186,3 +186,25 @@ fields:
       ]
     }
   url: https://hl7.org/fhir/appointment-definitions.html#Appointment.requestedPeriod
+
+- name: reasonCode
```

Comment:
there is already an entry for reasonCode on line 67, set its action field to a list of terminology and create

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/commit/063202ecf7b52c89b017b9b8d20412808d417975#diff-fbb215a73c336ca1ac34d91de2fb411eL70)